pkg/graveler/graveler.go

```diff
@@ -19,6 +21,14 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+var kvRetriesCounter = promauto.NewCounterVec(
+	prometheus.CounterOpts{
+		Name: "graveler_kv_retries",
+		Help: "Number of retries performed by Graveler to update value.",
+	},
+	[]string{"operation", "repository"},
```

Comment:
We shouldn't collect repos as metrics for 2 reasons:
1. Possible high cardinality
2. Possible user sensitive info. Again, not very likely but still.

It's not that any of the above is a high risk, but the gain of collecting the repos isn't big either. We don't collect repos anywhere else too.